autorest/adal/token.go

```diff
@@ -101,27 +100,35 @@ type Token struct {
 	AccessToken  string `json:"access_token"`
 	RefreshToken string `json:"refresh_token"`
 
-	ExpiresIn string `json:"expires_in"`
-	ExpiresOn string `json:"expires_on"`
-	NotBefore string `json:"not_before"`
+	ExpiresIn json.Number `json:"expires_in"`
```

Comment:
It may be a good moment to add some documentation about the expected contents of these values as well.

For example, I believe this is a whole number of seconds. We should say so :)

Reply:
I've added a link to the docs for this type.